src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts

```diff
@@ -126,4 +127,26 @@
     searchBox.val('');
     updateListedReviews();
   });
+
+  var prevLanguageValue = languageSelect.val();
+  languageSelect.on('change', function (e) {
+    var val = $(this).val();
+    switch (val) {
+      case 'C#':
+        val = "Csharp";
+        break;
+      case 'C++':
+        val = "Cpp";
+        break;
+    }
```

Comment:
```suggestion
    var val = $(this).val().replace("C++", "Cpp").replace("C#", "Csharp");
```

Would be a little more concise, if you want that.